src/components/view/pages/main/filters/filtersView.ts

```diff
@@ -0,0 +1,263 @@
+                  <div class="item-content__dual-range dual-range">
+                    <div class="slider"
+                         data-min_selected="${data.selected.size?.selectedMin || data.selected.size?.defaultMin}"
+                         data-max_selected="${data.selected.size?.selectedMax || data.selected.size?.defaultMax}"
+                         data-min="${data.size?.min}"
+                         data-max="${data.size?.max}"></div>
+                  </div>
+                </div>
+              </div>
+              <div class="filters__item filters-item">
+                <div class="filters-item__title">Category</div>
+                <div class="filters-item__content item-content">
+                  ${data.categories?.map((item) => {
+            const id = item.category.toLowerCase().replace(' ', '-')
+                    const checked = data.selected.categories?.some(c => c.category === item.category) ? 'checked="checked"' : '';
+                    // language=HTML
+                    return `<div class="item-content__category category">
+                      <label for="${id}" class="category__label">${item.category}</label>
+                      <div class="category__count">(${item.products})</div>
+                      <input id="${id}" type="checkbox" class="category__checkbox" data-categories="${item.category}" ${checked}>
+                  </div>`
+                  }).join('')}
+                </div>
+              </div>
+              <div class="filters__item filters-item">
+                <div class="filters-item__title">In stock</div>
+                <div class="filters-item__content item-content">
+                  <div class="item-content__stock stock">
+                    <input type="text" class="box-start" maxlength="2">
+                    <input type="text" class="box-end" maxlength="2">
+                  </div>
+                  <div class="item-content__dual-range dual-range">
+                    <div class="slider"
+                         data-min_selected="${data.selected.stock?.selectedMin || data.selected.stock?.defaultMin}"
+                         data-max_selected="${data.selected.stock?.selectedMax || data.selected.stock?.defaultMax}"
+                         data-min="${data.stock?.min}"
+                         data-max="${data.stock?.max}"></div>
+                  </div>
+                </div>
+              </div>
+              <div class="filters__close-btn"></div>
+            </div>
+        `
+    }
+
+    afterRender(controller: Controller) {
+        super.afterRender(controller);
+
+        const priceFilter = document.querySelector('.filters-item .price') as HTMLElement | null;
+        if (priceFilter) {
+            this.initializeSlider(priceFilter, (start, end) => {
+                Router.removeUrlParamKey('page');
+                Router.setUrlParam('minPrice', start)
+                Router.setUrlParam('maxPrice', end)
+            }, 2);
```

Comment:
I would suggest to avoid magic numbers, if you have `const` with meaningful name it will be better